app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -281,6 +281,15 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         })
     }
 
+    private fun recoverDaxDialogFromActivity(): Fragment? = activity?.supportFragmentManager?.findFragmentByTag(DAX_DIALOG_DIALOG_TAG)
```

Comment:
`recoverDaxDialogFromActivity` sounds like we are recovering the dialog after something happened based on something we are storing in the activity. But we are just finding the dialog by tag inside the fragmentManager. What about using a simple name as `get`/`find`/`obtain`DaxDialog? I don't think `FromActivity` is needed.

Reply:
Renamed to `getDaxDialogFromActivity`